src/components/ClassHeader.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+class ClassHeader extends React.Component {
+  render() {
+    return (
+      <header>
+        <h1>My first React App</h1>
+      </header>
+    );
+  }
+}
+
+export default ClassHeader;
```

Comment:
Старайся использовать экспорт именованый а не по-умолчанию. 
```suggestion
export ClassHeader;
```
В других проектах могут быть другие правила, но мы используем именованый экспорт/импорт.